src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -428,17 +475,18 @@ private static SyntaxList<StatementSyntax> InsertWithinTriviaOfNext(
         var precedingEndOfLine = nextStatementLeading.LastOrDefault(t => t.Kind() == SyntaxKind.EndOfLineTrivia);
         if (precedingEndOfLine == default)
         {
+            if (nextStatementLeading is [(kind: SyntaxKind.WhitespaceTrivia) indentation])
+                newStatement = newStatement.WithLeadingTrivia(indentation).WithTrailingTrivia(CarriageReturnLineFeed);
```

Comment:
>CarriageReturnLineFeed [](http://example.com/codeflow?start=94&length=22)

Does this get fixed up somewhere for NL based documents?

Reply:
oops.  no.  this should have been ElasticCRLF (which will get fixed up).